apps/www/src/lib/registry/new-york/example/PaginationDemo.vue

```diff
@@ -0,0 +1,58 @@
+<script setup lang="ts">
+import { ChevronLeftIcon, ChevronRightIcon, DoubleArrowLeftIcon, DoubleArrowRightIcon } from '@radix-icons/vue'
+import {
+  Pagination,
+  PaginationEllipsis,
+  PaginationFirst,
+  PaginationLast,
+  PaginationList,
+  PaginationListItem,
+  PaginationNext,
+  PaginationPrev,
+} from '@/lib/registry/new-york/ui/pagination'
+
+import {
+  Button,
+} from '@/lib/registry/new-york/ui/button'
+</script>
+
+<template>
+  <Pagination v-slot="{ page }" :total="100" :sibling-count="1" show-edges :default-page="2">
+    <PaginationList v-slot="{ items }" class="flex items-center gap-1">
+      <PaginationFirst as-child>
+        <Button class="w-9 h-9 p-0" variant="outline">
+          <DoubleArrowLeftIcon />
+        </Button>
```

Comment:
I think we can wrap these inside a wrapper component. As I observed, many of `shadcn-ui`'s icon were wrapped inside wrapper component.

I believe it would improve DX as dev just need to import the relevant component without needing to add the icon again.

WDYT?

Reply:
Correct 💯 

I'll do it

Also thanks for the invitation ❤️